Refine backup and restore documentation (#518)
pingcap/tidb-operator: docs/operation-guide.md

$ stern -n ${namespace} tidb -c slowlog
```

> ## Backup and Restore

Reviewer: ```suggestion
## Backup and restore
```

> TiDB Operator provides highly automated backup and recovery operations for TiDB cluster. You can easily take full backup or setup incremental backup of TiDB cluster, and restore the TiDB cluster when the cluster fails.

Reviewer: ```suggestion
TiDB Operator provides highly automated backup and recovery operations for a TiDB cluster. You can easily take full backup or setup incremental backup of a TiDB cluster, and restore the TiDB cluster when the cluster fails.
```

> For detail operation guides of backup and restore, please refer to [Backup and Restore TiDB Cluster](./backup-restore.md).

Reviewer: ```suggestion
For detailed operation guides of backup and restore, refer to [Backup and Restore TiDB Cluster](./backup-restore.md).
```
